src/XRP/model/account-set-flag.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Flags used in AccountSet transactions.
+ *
+ * @see https://xrpl.org/accountset.html#accountset-flags
+ */
+export enum AccountSetFlag {
```

Comment:
Small note, but I really appreciate how well commented these enums are in Xpring JS and that we are referencing the Xrpl docs. I wish the table in the docs displayed these flags in order too but thats outside of our scope.